extensions/amp-story/1.0/amp-story-page.js

```diff
@@ -302,6 +302,14 @@ export class AmpStoryPage extends AMP.BaseElement {
     /** @private @const {!../../../src/service/timer-impl.Timer} */
     this.timer_ = Services.timerFor(this.win);
 
+    const audioDeferred = new Deferred();
+
+    /** @private @const {!Promise} */
+    this.backgroundAudioUpgradePromise_ = audioDeferred.promise;
```

Comment:
Nit: we could only have one property that's the deferred and then call .promise/.resolve()?